### DISCUSSION TITLE
Can i redirect the port to a different IP without the host? I need to access IP:PORT.

### DISCUSSION THREAD

**Discussion by FullSilence** [hassio-addons/addon-nginx-proxy-manager#477] [Q&A]
Can you please tell me how to do the following?

location / {
proxy_pass http://192.168.1.63:9000/;
proxy_set_header Host $host;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
}

telnet 'MY_PUBLIC_IP' 9000

**Comment by frenck** [comment]
As I told you in the issue as well. Nope. You can't.

../Frenck
